packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -13,7 +13,7 @@ import { DirectionalHint } from '../../common/DirectionalHint';
 
 const getClassNames = classNamesFunction<ITooltipStyleProps, ITooltipStyles>();
 
-@customizable('Tooltip', ['theme'])
+@customizable('Tooltip', ['theme', 'styles'])
```

Comment:
This change is the least intrusive, but it's also very repetitive. Would it make sense to default the customizable props to them and styles?